pkg/common/common.go

```diff
@@ -18,43 +17,14 @@ const (
 	OADPOperator               = "oadp-operator"
 	OADPOperatorVelero         = "oadp-operator-velero"
 	OADPOperatorServiceAccount = "openshift-adp-controller-manager"
+	OperatorTypeMTC            = "mtc"
 )
 
-var DefaultRestoreResourcePriorities = restore.Priorities{
-	HighPriorities: []string{
-		"securitycontextconstraints",
-		"customresourcedefinitions",
-		"namespaces",
-		"roles",
-		"rolebindings",
-		"clusterrolebindings",
-		"managedcluster.cluster.open-cluster-management.io",
-		"managedcluster.clusterview.open-cluster-management.io",
-		"klusterletaddonconfig.agent.open-cluster-management.io",
-		"managedclusteraddon.addon.open-cluster-management.io",
-		"storageclasses",
-		"volumesnapshotclass.snapshot.storage.k8s.io",
-		"volumesnapshotcontents.snapshot.storage.k8s.io",
-		"volumesnapshots.snapshot.storage.k8s.io",
-		"datauploads.velero.io",
-		"persistentvolumes",
-		"persistentvolumeclaims",
-		"serviceaccounts",
-		"secrets",
-		"configmaps",
-		"limitranges",
-		"pods",
-		"replicasets.apps",
-		"clusterclasses.cluster.x-k8s.io",
-		"endpoints",
-		"services",
-	},
-	LowPriorities: []string{
-		"clusterbootstraps.run.tanzu.vmware.com",
-		"clusters.cluster.x-k8s.io",
-		"clusterresourcesets.addons.cluster.x-k8s.io",
-	},
-}
+// Labels
+const (
+	OadpOperatorLabel       = "openshift.io/oadp"
```

Comment:
Can you move this outside package common? I think this would make importing github/oadp/pkg/common add a lot of indirect imports for QE if they are using this.

Could be pkg/common/labels

Reply:
api package via _types.go was nice cause imports there is minimal

Reply:
I think it has more imports than this file, but creating something like `Common/consts.go` (and `Common/functions.go`) would make this file have no imports

Reply:
imports are based on package level too.. so it has to be a different subdir

Reply:
common/consts/labels.go